commit 45066b87374d005fa25865bee24a8342b564397a (refs/changes/11/1211/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-01-06T01:13:18+00:00 (2 years, 9 months ago)
    
    MANTA-3075 muskie should bump cueball version for perf improvements
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 3bdaebd..cbf77f6 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "bunyan-syslog": "0.2.2",
-        "cueball": "1.2.2",
+        "cueball": "1.3.0",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
